packages/web-components/fast-components/src/horizontal-scroll/horizontal-scroll.styles.ts

```diff
@@ -0,0 +1,123 @@
+import { css } from "@microsoft/fast-element";
+import { DirectionalStyleSheetBehavior } from "@microsoft/fast-foundation";
+
+const ltrActionsStyles = css`
+    .scroll-prev {
+        right: auto;
+        left: 0;
+    }
+
+    .scroll-next:before {
+        left: auto;
+        right: 0;
+        background: linear-gradient(to right, transparent, var(--scroll-fade-next));
+    }
```

Comment:
Should there be an instance of scroll-previous as well for LTR?

Reply:
.scroll:before in the base styles act as the .scroll-prev:before for LTR and .scroll-next:before for RTL. It's easier to think of this as .scroll.scroll-next:before which overrides some of the styles used by .scroll.scroll-prev:before. If that makes sense.

Reply:
Yeah, I missed this - that's all :)